sanic/handlers.py

```diff
@@ -25,7 +26,9 @@ class ErrorHandler:
     """
 
     # Beginning in v22.3, the base renderer will be TextRenderer
-    def __init__(self, fallback: str, base: Type[BaseRenderer] = HTMLRenderer):
+    def __init__(
+        self, fallback: str = "auto", base: Type[BaseRenderer] = HTMLRenderer
```

Comment:
Type annotation for `fallback` is no longer needed.

Reply:
But also is not harmful.